src/test/scala/io/iohk/ethereum/consensus/pow/validators/EthashBlockHeaderValidatorSpec.scala

```diff
@@ -254,6 +254,36 @@ class EthashBlockHeaderValidatorSpec
     difficulty shouldBe afterRewardReductionBlockHeader.difficulty
   }
 
+  it should "properly calculate the difficulty after muir glacier delay" in new EphemBlockchainTestSetup {
+    val blockchainConfigWithoutDifficultyBombRemoval: BlockchainConfig =
+      blockchainConfig.withUpdatedForkBlocks(
+        _.copy(
+          difficultyBombRemovalBlockNumber = BigInt("1000000000000"),
+          difficultyBombPauseBlockNumber = 0,
+          difficultyBombContinueBlockNumber = 0,
+          muirGlacierBlockNumber = 9200000
+        )
+      )
+
+    val parentHeader: BlockHeader =
+      validParentBlockHeader.copy(
+        number = 9200000 - 1,
+        unixTimestamp = 1525176000,
+        difficulty = BigInt("22627021745803")
+      )
+    val parent = Block(parentHeader, parentBody)
+
+    val blockNumber: BigInt = parentHeader.number + 1
+    val blockTimestamp: Long = parentHeader.unixTimestamp + 6
+
+    val difficulty: BigInt = EthashDifficultyCalculator.calculateDifficulty(blockNumber, blockTimestamp, parent.header)(
+      blockchainConfigWithoutDifficultyBombRemoval
+    )
+    val blockDifficultyWihtoutBomb = BigInt("22638070096265")
```

Comment:
For the sake of completness, are those numbers taken from somewhere ?

Reply:
Not really, they have to be calculated manually. Without the delay this number is orders of magnitude bigger